AsyncDisplayKit/ASMapNode.mm

```diff
@@ -307,6 +318,15 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   return size;
 }
 
+- (void)calculatedLayoutDidChange
+{
+  [super calculatedLayoutDidChange];
+  
+  if (_snapshotNeeded) {
+    [self takeSnapshot];
```

Comment:
This also degrades the functionality of ASMapNode to only take a snapshot on a layout change. Whereas we may want to take a new snapshot without a layout change.